test/build.bats

```diff
@@ -127,3 +127,10 @@ load common
     ldd sotest/sotest
     sotest/sotest
 }
+
+
+@test 'squashfs ready' {
```

Comment:
It was the first baby test I wrote. It is no longer needed. Perhaps someday if the squashfs were to become mandatory it would have a purpose. But today is not that day.